doc/rados/operations/stretch-mode.rst

```diff
@@ -141,17 +139,36 @@ your CRUSH map. This procedure shows how to do this.
 
 #. Run the monitors in connectivity mode. See `Changing Monitor Elections`_.
 
+   .. prompt:: bash $
+
+      ceph mon set election_strategy connectivity
+
 #. Command the cluster to enter stretch mode. In this example, ``mon.e`` is the
    tiebreaker monitor and we are splitting across data centers. The tiebreaker
    monitor must be assigned a data center that is neither ``site1`` nor
-   ``site2``. For this purpose you can create another data-center bucket named
-   ``site3`` in your CRUSH and place ``mon.e`` there:
+   ``site2``. This data center **should not** be defined in your CRUSH map, here 
+   we are placing ``mon.e`` in a virtual data center called ``site3``:
 
    .. prompt:: bash $
 
       ceph mon set_location e datacenter=site3
       ceph mon enable_stretch_mode e stretch_rule datacenter
 
+#. Set the replication levels for each pool. Here we are setting the standard
```

Comment:
@PC-Admin how did you end up manually setting sizes? Enabling stretch mode automatically does this for every pool which has the default configuration.

Reply:
I just assumed if it wasn't in the CRUSH rules I'd have to do it myself. If it's done automatically then I'll go ahead and remove this section.

Reply:
I'll merge and backport this now.